modules/restful_example/src/Plugin/resource/Articles__1_0.php

```diff
@@ -39,17 +41,55 @@ class Articles__1_0 extends ResourceEntity implements ResourceInterface {
   protected function publicFields() {
     return array(
       'id' => array(
-        'wrapper_method' => 'getIdentifier',
-        'wrapper_method_on_entity' => TRUE,
+        'property' => 'nid',
       ),
       'label' => array(
         'wrapper_method' => 'label',
         'wrapper_method_on_entity' => TRUE,
+        'process_callbacks' => array(
+          array(array($this, 'addPrefix'), array('Label: ')),
+        ),
       ),
       'self' => array(
         'callback' => array($this, 'getEntitySelf'),
       ),
+      'tags' => array(
+        'property' => 'field_tags',
+        'resource' => array(
+          'name' => 'tags',
+          'majorVersion' => 1,
+          'minorVersion' => 0,
+        ),
+      ),
+      'status' => array(
+        'property' => 'status',
+        'methods' => array(RequestInterface::METHOD_POST, RequestInterface::METHOD_PUT),
```

Comment:
Myabe it's a bad example, I just wanted to show that it won't appear for GET requests.